### PR TITLE
update solution cisco asa with data connector

### PR DESCRIPTION
**Pull request by v-laanjana**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-amolpatil** [comment]
Please update `Azure Sentinal` to `Microsoft Sentinel`. there is miss spell of `Sentinal`
image

File: Solutions/CiscoASA/Data Connectors/CiscoASA.JSON [hunk 1]
```diff
@@ -0,0 +1 @@
... if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python --version\n\n>2. You must have elevated permissions (sudo) on your machine","instructions":[{"parameters":{"fillWith":["WorkspaceId"],"label":"Run the following command to validate your connectivity:","value":"sudo wget -O cef_troubleshoot.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"},"type":"CopyableLabel"}]},{"title":"4. Secure your machine ","description":"Make sure to configure the machine's security according to your organization's security policy\n\n\n[Learn more >](https://aka.ms/SecureCEF)"}],"metadata":{"id":"31e95c16-9972-4afe-9697-05b9f40f2a58","version":"1.0.0","kind":"dataConnector","source":{"kind":"community"},"author":{"name":"Cisco Systems"},"support":{"name":"Cisco Systems","link":"https://www.cisco.com/c/en/us/support/index.html","tier":"developer"}}}
```
Comment (v-amolpatil) [inline Solutions/CiscoASA/Data Connectors/CiscoASA.JSON:1]: Please fix the formatting of this file content. Complete file Json is showing in a single line.

File: Solutions/CiscoASA/Playbooks/CiscoASA-AddIPtoNetworkObjectGroup/readme.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ When a new Sentinel incident is created,this playbook gets triggered and perform
 
 
 ### Prerequisites
-1. **This playbook template is based on Azure Sentinel Incident Trigger which is currently in Private Preview (Automation Rules).** You can change the trigger to the Sentinel Alert trigger in cases you are not part of the Private Preview.
+1. **This playbook template is based on microsoft sentinel Incident Trigger which is currently in Private Preview (Automation Rules).** You can change the trigger to the Sentinel Alert trigger in cases you are not part of the Private Preview.
```
Comment (v-amolpatil) [inline Solutions/CiscoASA/Playbooks/CiscoASA-AddIPtoNetworkObjectGroup/readme.md:26]: Please caps initials letter of microsoft and sentinel work. eg: Microsoft Sentinel.

File: Solutions/CiscoASA/Data Connectors/CiscoASA.JSON (hunk 1 above)
Comment (v-laanjana) [reply Solutions/CiscoASA/Data Connectors/CiscoASA.JSON]: done

**Comment by v-amolpatil** [comment]
Please update spelling mistake as shown below in playbook file and maintemplate file
image

**Review by v-amolpatil — APPROVED** [review]

**Pull request merged**